src/ShadowChild.tsx

```diff
@@ -18,14 +19,18 @@ const ShadowChild: React.FC<Props> = ({
   hideAfter,
   highlightStyle,
   clearHandler,
-  updateClearHandler
+  updateClearHandler,
+  uniqueId
 }) => {
   const initialMount = useRef(true);
   const [clearHighlightRef, setClearHighlightRef] = useState(clearHandler);
   const changedChildren = Array.isArray(child.props.children)
     ? child.props.children.find(c => c.toString().trim().length)
     : child.props.children;
 
+  const elementDOM = child.ref.current;
+  elementDOM.setAttribute('react-change-hightlight-uniqueId',uniqueId);
```

Comment:
you can name it a bit shorter as the default class name `rc-highlight-id` which make it simple and we recogise it as the uniqueId of course